scripts/mlp/centroidal/topt.py

```diff
@@ -296,21 +296,21 @@ def generateCentroidalTrajectory(cs,cs_initGuess = None,fullBody=None, viewer =N
     
     #add all effector phases to the problem : 
     i = 0
-    for ee in effectors_phases.keys():
+    for ee in list(effectors_phases.keys()):
```

Comment:
This looks like a `2to3` useless thing.

`2to3` thinks that `.keys()` method were returning lists in python 2, and are now returning iterators in python 3 ; so if you really don't want to change your behavior, you have to escape your iterator in a list.

But actually, here, an iterator is better than a list :)

Reply:
Ok, changes done.